packages/homepage/src/templates/docs.js

```diff
@@ -263,33 +264,32 @@ export default class Docs extends React.Component {
 
 export const pageQuery = graphql`
   query Docs($slug: String!) {
-    allMarkdownRemark(
-      filter: { fields: { slug: { regex: "/docs/" } } }
-      sort: { fields: [fileAbsolutePath], order: ASC }
+    allDocs: allMarkdownRemark(
+      filter: { fileAbsolutePath: { regex: "/docs/" } }
+      sort: { fields: [fileAbsolutePath], order: [ASC] }
```

Comment:
What is the difference between `[ASC]` and `ASC`?

Reply:
No real difference in this case, just a bit cleaner to specify it for each sorted field I think 🙂